tensorflow/lite/core/model_builder.cc

```diff
@@ -90,6 +100,124 @@ std::unique_ptr<FlatBufferModel> FlatBufferModel::VerifyAndBuildFromBuffer(
                                       error_reporter);
 }
 
+#if FLATBUFFERS_LITTLEENDIAN == 0
+
+void FlatBufferModel::ByteSwapSerializedModel(std::string* serialized_model){
+  const uint8_t* buffer =
+      reinterpret_cast<const uint8_t*>(serialized_model->c_str());
+  const tflite::Model* input_model = tflite::GetModel(buffer);
+  ByteSwapTFLiteModel(input_model);
+}
+
+void FlatBufferModel::ByteSwapBuffer(int8_t tensor_type, size_t buffer_size,
+    uint8_t* buffer){
+  switch (tensor_type) {
+    // 16-bit types
+    case tflite::TensorType_FLOAT16:
+    case tflite::TensorType_INT16:
+    case tflite::TensorType_UINT16: {
+      auto bp = reinterpret_cast<uint16_t*>(buffer);
+      for (int i = 0; i < buffer_size/2; i++)
+        bp[i] = flatbuffers::EndianSwap(bp[i]);
+      break;
+    }
+    // 32-bit types
+    case tflite::TensorType_FLOAT32:
+    case tflite::TensorType_INT32:
+    case tflite::TensorType_UINT32:
+    case tflite::TensorType_COMPLEX64: {
+      auto bp = reinterpret_cast<uint32_t*>(buffer);
+      for (int i = 0; i < buffer_size/4; i++)
+        bp[i] = flatbuffers::EndianSwap(bp[i]);
+      break;
+    }
+    // 64-bit types
+    case tflite::TensorType_INT64:
+    case tflite::TensorType_FLOAT64:
+    case tflite::TensorType_UINT64:
+    case tflite::TensorType_COMPLEX128: {
+      auto bp = reinterpret_cast<uint64_t*>(buffer);
+      for (int i = 0; i < buffer_size/8; i++)
+        bp[i] = flatbuffers::EndianSwap(bp[i]);
+      break;
+    }
+    default:
+      break;
+  }
+}
+
+void FlatBufferModel::ByteSwapTFLiteModel(const tflite::Model* tfl_model){
```

Comment:
Could you run formatter? https://www.tensorflow.org/community/contribute/code_style#c_coding_style

Need a space before '{'

Reply:
I've run the formatter and updated the code. Thanks!